evm/src/eNear/contracts/ENearProxy.sol

```diff
@@ -53,6 +58,22 @@ contract ENearProxy is UUPSUpgradeable, AccessControlUpgradeable, ICustomMinter
         eNear.transferToNear(amount, string(''));
     }
 
+    function finaliseNearToEthTransfer(
+        bytes memory proofData,
+        uint64 proofBlockHeight
+    ) external whenNotPaused(PAUSED_LEGACY_FIN_TRANSFER) {
+        require(
+            prover.proveOutcome(proofData, proofBlockHeight),
+            "Proof should be valid"
+        );
+
+        eNear.finaliseNearToEthTransfer(proofData, proofBlockHeight);
+    }
+
+    function pause(uint flags) external onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
Let's also add `pauseAll` with pause manager role

Reply:
61aaa7c67d6d76789e1f8ed216776b4e34f80681

Reply:
We should unify the naming of the role between our EVM contracts.
https://github.com/Near-One/omni-bridge/blob/main/evm%2Fsrc%2Fomni-bridge%2Fcontracts%2FOmniBridge.sol#L37

Reply:
Done